docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: '3'
+services:
+  uajp:
+    build: .
+    tty: true
+    depends_on:
+      - mysql_db
+  
```

Comment:
Our application uses port of `3000`. So, you should expose `3000` to `localhost`.